exercises/collatz-conjecture/src/test/scala/CollatzConjectureTest.scala

```diff
@@ -0,0 +1,28 @@
+import org.scalatest.{Matchers, FunSuite}
+
+class CollatzConjectureTest extends FunSuite with Matchers {
+
+  test("zero steps for one") {
+    pending
+    CollatzConjecture.collatz(1) should be (0)
+  }
+
+  test("divide if even") {
+    pending
+    CollatzConjecture.collatz(16) should be (4)
+  }
+
+  test("even and odd steps") {
+    pending
+    CollatzConjecture.collatz(12) should be (9)
+  }
+
+  test("Large number of even and odd steps") {
+    pending
+    CollatzConjecture.collatz(1000000) should be (152)
+  }
+
+  test("Large number of even and odd steps") {
+    pending
+    CollatzConjecture.collatz(1000000) should be (152)
+  }
```

Comment:
Missing final "}". This class does not compile.
I am actually surprised that the travis CI build did not error.

As a note - we are going through the process of creating test code generators to take the exercism canonical-data.xml file and generate the unit tests. No big deal in this case. Ultimately I can build the generator for this exercise once I get some time. For now, hand generated unit tests are fine. If you are interested, the test generators are in the scala/testgen directory.